ddtrace/tracer/log.go

```diff
@@ -107,6 +109,11 @@ func logStartup(t *tracer) {
 	if _, err := samplingRulesFromEnv(); err != nil {
 		info.SamplingRulesError = fmt.Sprintf("%s", err)
 	}
+	// Based on the apply logic in sampler.go func (*rulesSampler).applyRate,
+	// the rate limit applies only when there are rules or there is a global rate set.
+	if t.rulesSampling.globalRate > 0 || len(t.rulesSampling.rules) > 0 {
+		info.SampleRateLimit = fmt.Sprintf("%f", t.rulesSampling.limiter.limiter.Limit())
```

Comment:
Definitely. I don't think it can be `nil` right now, but that's almost impossible to verify and could easily change in the future. I'll add some `nil` checks here.